css/styles.css

```diff
@@ -0,0 +1,177 @@
+html, div, ul, ol {
```

Comment:
I think you're better off reversing what you're ignoring and pushing up. If someone was collaborating on this code, they would need the SASS file, not the compiled CSS, which would be generated by running your Grunt task.